recipe/meta.yaml

```diff
@@ -50,32 +46,30 @@ outputs:
         - setuptools
         - wheel
         - pillow
-        - numpy
+        - numpy {{ numpy }}
       run:
         - python
         - filelock
-        - huggingface_hub >=0.13.2
+        - huggingface_hub >=0.23.2
         - importlib-metadata
         - numpy
         - pillow
         - regex !=2019.12.17
         - requests
+        - safetensors >=0.3.1
```

Comment:
Can you add a run_constrained section following thse links: 
https://github.com/huggingface/diffusers/blob/v0.30.3/src/diffusers/dependency_versions_table.py/
https://github.com/huggingface/diffusers/blob/v0.30.3/setup.py#L211
https://github.com/huggingface/diffusers/blob/v0.30.3/setup.py#L231

Reply:
Done